packages/storage-driver-s3/src/index.ts

```diff
@@ -279,12 +284,12 @@ export class DriverS3 implements TusDriver {
 			...(context.metadata?.['contentType']
 				? {
 						ContentType: context.metadata['contentType'],
-				  }
+					}
```

Comment:
My VSCode is formatting this as tabs instead of spaces.
But why we have spaces in first place? I am rendering space characters and I only see tabs to ident, not spaces 🤔

Reply:
Hmm... No idea why, but I have attempted to fix the linting error via https://github.com/directus/directus/pull/24036/commits/29bc3e7affcc5a02a23c6e2fd4cf9e05723ec14b

Reply:
I have enabled ESLint format on the extension in VS Code and now it's working as expected 👍